org.gabmus.razercommander.json

```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "org.gabmus.razercommander",
+    "command": "razercommander",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "razercommander",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--share=network",
+
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.razer",
+
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=~/.config/razercommander:rw",
```

Comment:
This shouldn't be needed.

Reply:
it is, that's where the configuration files are saved

Reply:
All applications have a private `$XDG_CONFIG_HOME`, that is where you save configs. Are you saying the daemon also uses that directory and you have to modify it there?

Reply:
> Are you saying the daemon also uses that directory and you have to modify it there?

No, it doesn't

Alright, so I *can* let razercommander write into its own `$XDG_CONFIG_HOME`, but if I let it use the real `~/.config` folder, the flatpak package will share the same configuration of the regular package. This way users won't have to re-configure profiles and macros for their devices if they migrate to the flatpak version.

This said, I'd prefer to keep this as it is right now, but if using the private `$XDG_CONFIG_HOME` is required to get razerCommander accepted in flathub, I will change it.

Reply:
It should be using a private directory, the entire point of flatpak is being self contained.

Reply:
alright, will change that